src/components/multi-autocompleter/multi-autocompleter.jsx

```diff
@@ -93,9 +94,10 @@ MultiAutocompleter.propTypes = {
 
 MultiAutocompleter.defaultProps = {
   className: multiSelectStyles.container,
+  extraParams: '',
   onChange: () => {},
   optionIDGetter: option => option.id,
-  optionLabelGetter: option => option.title || option.name || option.full_name || option.currency,
+  optionLabelGetter: option => option.title || option.name || option.full_name || option.currency || option.label,
```

Comment:
It might be worthwhile to add an inline comment in the prop-type for `optionLabelGetter`.

```suggestion
  /** A function to get the label of an option. Defaults to accessing one of `title`, `name`, `full_name`, `currency`, or `label` keys.  */
  optionLabelGetter: PropTypes.func,```

Reply:
I'm not sure if we wanted test coverage on the new `option.label` bit.

Reply:
Test added, and also support for `model.label` was backfilled to `Autocompleter`.